azkaban_cli/azkaban_cli.py

```diff
@@ -271,6 +272,50 @@ def __change_permission(ctx, project, group, admin, read, write, _execute, _sche
     except ChangePermissionError as e:
         logging.error(str(e))
 
+def __parse_flow_execution(json):
```

Comment:
maybe just a matter of opinion, but i would prefer something like __log_flow_execution, I think it is more suggestive

Reply:
Fine by me. Fixed in new commit.